cli/tests/pcluster/validators/test_awsbatch_validators.py

```diff
@@ -358,3 +359,15 @@ def test_is_instance_type_format(candidate, expected_return_value):
     assert_that(AwsBatchInstancesArchitectureCompatibilityValidator._is_instance_type_format(candidate)).is_equal_to(
         expected_return_value
     )
+
+
+@pytest.mark.parametrize(
+    "scheduler, expected_message",
+    [
+        ("slurm", None),
+        ("awsbatch", "FSx for Lustre is not supported when using AWS Batch as scheduler"),
+    ],
+)
+def test_awsbatch_region_validator(scheduler, expected_message):
```

Comment:
`test_awsbatch_region_validator` should be rename to `test_fsx_scheduler_validator`